tests/unit/integrations/mlflow/experiment_trackers/test_mlflow_experiment_tracker.py

```diff
@@ -107,9 +109,9 @@ def test_mlflow_experiment_tracker_stack_validation() -> None:
         created=datetime.now(),
         updated=datetime.now(),
     )
-    remote_artifact_store = GCPArtifactStore(
+    remote_artifact_store = S3ArtifactStore(
         name="",
-        config=GCPArtifactStoreConfig(path="gs://my-bucket"),
+        config=S3ArtifactStoreConfig(path="s3://tmp"),
```

Comment:
how is the flavor (below) gcp if the config has been changed to s3?

Reply:
Nice catch, I changed flavor to "s3" accordingly. Didn't seem to have any effect actually.